### DISCUSSION TITLE
Jules, OpenSMTPD 6.6.0 upcoming release and related things · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#107] [Articles]

**Comment by whataboutpereira** [comment]
Oh, congrats on forking Jules! :D My neighbour had his firstborn just 3 weeks ago as well. :D

**Comment by poolpOrg** [comment]
Thanks :-)

**Comment by hrkfdn** [comment]
First of all, congratulations!
Thank you for your work on smtpd and these reports, I enjoy reading them a lot.

**Comment by poolpOrg** [comment]
Thank you :-)
